src/12-fine-tuning/lib/test.ts

```diff
@@ -0,0 +1,71 @@
+import OpenAI from "openai"
+
+// spell-checker:disable
+const testCases = {
+  "Oil on canvas": "Painting",
+  "Stainless steel, painted wood": "Sculpture",
+  "Screenprint in colors on Cream Speckletone paper": "Print",
```

Comment:
Small and not very representative dataset, probably lowers the success rate artificially (to about 75%). I suspect a more realistic test would do better, but just wanted to wire everything up here rather than prove how good this model is.